src/components/o-testimonials/_preview.html

```diff
@@ -0,0 +1,24 @@
+<axa-testimonials classes="o-testimonials"
+                  title="Customer Reviews"
+                  auto-rotate-enabled="true"
+                  auto-rotate-time="5000"
+                  subtitle="AXA works hard to provide the best service possible to its customers.">
+  <div class="o-testimonials__item js-o-testimonial__item">
```

Comment:
should be a `<p>` tag

Reply:
@nakovskijosif 
We intend to avoid the need of component related classes within the **light DOM**.

Every component should just take in data or other non-related components and produce a nice DOM tree.
Please refactor this and only use `<div>` s if they make semantically sense.

Reply:
I would say, yes andy but also sometimes it can make sense. So i would say is ok